### PR TITLE
[4.0] [mod_status] Improvements

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

- Use namespaced classes everywhere

### CODE REVIEW

**Comment by ghost** [comment]
After PR applied got on all Menus so install new one:
bildschirmfoto 2017-12-07 um 14 24 51

**Comment by C-Lodder** [comment]
@franz-wohlkoenig oops, fixed.

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 6508ee104bee409ec6e3308fdbfc7202670384f1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18986">issues.joomla.org/tracker/joomla-cms/18986</a>.</sub>

File: administrator/modules/mod_status/tmpl/default.php [hunk 1]
```diff
@@ -67,20 +76,20 @@
...
 				</div>
 				<?php $route = 'index.php?option=com_admin&amp;task=profile.edit&amp;id=' . $user->id; ?>
-				<a class="dropdown-item" href="<?php echo JRoute::_($route); ?>">
-					<?php echo JText::_('MOD_STATUS_EDIT_ACCOUNT'); ?></a>
-				<a class="dropdown-item" href="<?php echo JRoute::_('index.php?option=com_login&task=logout&'
-					. JSession::getFormToken() . '=1') ?>"><?php echo JText::_('JLOGOUT'); ?></a>
+				<a class="dropdown-item" href="<?php echo Route::_($route); ?>">
+					<?php echo Text::_('MOD_STATUS_EDIT_ACCOUNT'); ?></a>
+				<a class="dropdown-item" href="<?php echo Route::_('index.php?option=com_login&task=logout&'
+					. Session::getFormToken() . '=1') ?>"><?php echo Text::_('JLOGOUT'); ?></a>
```
Comment (Quy) [inline administrator/modules/mod_status/tmpl/default.php]: Add semicolon after `'=1')`

**Pull request merged**